packages/e2e-tests/run-e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+cd ../devtools
+yarn start:shared-ganache & 
+ganache=$!
+
+cd ../$1 && yarn run wait-on ../../.ganache-deployments/ganache-deployments-8545.json
+yarn start &
+app=$!
+
+cd ../xstate-wallet
+yarn start &
+wallet=$!
+
+cd ../simple-hub
+yarn hub:watch &
+hub=$!
```

Comment:
I decided to run the `hub` every time, even for e2e tests that do not require it.

Related issue: I don't think we can `wait-on` the hub to be ready, although we probably want to do that. There might be a workaround?

Reply:
Probably we should add a `lock.pid` file which has the process ID and get deleted when the hub shuts down.

Reply:
Would that help us wait for the hub to be 'ready'?

Reply:
Hm, no I suppose not. I guess there are two viable options:
1. Have a message type that the hub is expected to reply to. Like `{ recipient: "firebase:simple-hub", value: "ping" }` that gets a reply `"pong"`. Then have a little script which pings until it gets a pong.
2. Add a very lightweight HTTP server and add a `/heartbeat` endpoint that returns `"OK"` when everything is "ready"
I'd say we should do #2 (in a separate PR) as it relates to an item on my [wishlist](https://www.notion.so/elfour/0ee12a01ea684be095ca3b38909c9399?v=282f6c575af544e88924e7a1864a7a57&p=87f9f19a1da04c89849829d79920ac10)